extensions/jms/src/main/java/org/mule/extensions/jms/internal/connection/provider/activemq/ActiveMQConnectionFactoryProvider.java

```diff
@@ -77,7 +76,7 @@ ConnectionFactory createDefaultConnectionFactory() throws ActiveMQConnectionExce
       String message = format("Failed to create a default Connection Factory for ActiveMQ using the [%s] implementation: ",
                               getFactoryClass());
       LOGGER.error(message, e);
-      throw new ActiveMQConnectionException(createStaticMessage(message), e);
+      throw new ActiveMQException(message, e);
```

Comment:
I think `ActiveMQConnectionException` communicates more clearly the context of the problem than just `ActiveMQException`. Is a "ConnectionException" since it directly relates to the JmsConnection